assets/js/components/TourTooltips.test.js

```diff
@@ -452,4 +463,29 @@ describe( 'TourTooltips', () => {
 			);
 		} );
 	} );
+
+	it( 'should set stepIndex to 0 when tour is started and step key is undefined', async () => {
+		registry.dispatch( CORE_UI ).setValue( STEP_KEY, undefined );
+
+		const { getByRole } = renderTourTooltipsWithMockUI( registry );
+		await getByRole( 'alertdialog' );
+		mockTrackEvent.mockClear();
+
+		// Go to step 2
+		fireEvent.click( getByRole( 'button', { name: /next/i } ) );
+		await getByRole( 'alertdialog' );
+		// Tracks the advance on the 1st step, view on the 2nd step.
+		expect( mockTrackEvent ).toHaveBeenNthCalledWith(
+			1,
+			EVENT_CATEGORY,
+			GA_ACTIONS.NEXT,
+			1
+		);
+		expect( mockTrackEvent ).toHaveBeenNthCalledWith(
+			2,
+			EVENT_CATEGORY,
+			GA_ACTIONS.VIEW,
+			2
+		);
+	} );
```

Comment:
I would suggest we remove this, as:
1. The test case doesn't really verify the step index in its contents.
2. We're essentially duplicating the test case. We have similar (if not the same) verifications in the test cases above.

WDYT?

```suggestion
```